e2e/login/login.feature

```diff
@@ -0,0 +1,17 @@
+# Created by jludvice at 1.3.17
+@login
+Feature: First pass at login, homepage, connections
```

Comment:
This user story and narrative approach using Cucumber is really nice and fits well with agile methodologies.